curses.c

```diff
@@ -276,12 +277,25 @@ int mtr_curses_keyaction(void)
     switch ( mtrtype ) {
     case IPPROTO_ICMP:
     case IPPROTO_UDP:
+    case IPPROTO_SCTP:
       mtrtype = IPPROTO_TCP;
       break;
     case IPPROTO_TCP:
       mtrtype = IPPROTO_ICMP;
       break;
     }
+  }
+  if (tolower(c) == 's') {
+    switch ( mtrtype ) {
+    case IPPROTO_ICMP:
+    case IPPROTO_UDP:
+    case IPPROTO_TCP:
+      mtrtype = IPPROTO_SCTP;
+      break;
+    case IPPROTO_SCTP:
+      mtrtype = IPPROTO_ICMP;
+      break;
+    }
     return ActionNone;
```

Comment:
Oh, it will surely not work. `s` conflicted with line 135, the key to `Change Packet Size`.